.github/workflows/array_api.yml

```diff
@@ -111,6 +111,12 @@ jobs:
     - name: Test SciPy
       run: |
         export OMP_NUM_THREADS=2
+        export SCIPY_DEFAULT_DTYPE="float32"
```

Comment:
Thanks for looking into this! Do you think the default dtypes should be the other way around  in the ci job so that all the float64 tests are together and then the separate run is for float32 default?